leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -69,6 +69,14 @@ var ElementInspector = Backbone.View.extend({
         this.listenTo(this, 'change: intention', this.initSatValueChanged); 
         // Saves this.model.get('intention) as a local variable to access it more easily
         this.intention = this.model.get('intention');
+        this.satValueDict = {
```

Comment:
Is this second dictionary required?

Reply:
Not if we replace the values in the option tags with their binary string equivalents, but I would need to go through further to determine if that would make things easier/more confusing. Happy to do that here or in a future PR